ui-tests-protractor/e2e/conf/protractor.base.conf.js

```diff
@@ -5,25 +5,43 @@ let reporter = require('cucumber-html-reporter');
 let fse = require('fs-extra');
 let path = require('path');
 const os = require('os');
+let merge = require('merge');
 
 // define paths for cucumber test framework
 let testPath = path.resolve('e2e/');
 let reportsPath = path.join(testPath, 'cucumber-reports/');
 let cucumberJsonPath = path.join(reportsPath, 'cucumber-report.json');
 let cucumberHtmlPath = path.join(reportsPath, 'cucumber-report.html');
 
+let useBrowser = process.env.BROWSER || null;
 
-exports.config = {
+if (useBrowser === `firefox`) {
+
+  console.log(`Using firefox browser.`);
+  config = { 
+    capabilities: {
+      'browserName': 'firefox'
+    }
+  };
+  
+} else {
+  
+  console.log(`Using chrome as default browser. Run \`export BROWSER=firefox\` to use firefox.`);  
```

Comment:
And one more... Maybe proper capitalization for browser names. My OCD kicks-in sorry :)